slangpy/types/buffer.py

```diff
@@ -100,6 +104,62 @@ def resolve_element_type(program_layout: SlangProgramLayout, element_type: Any)
     return element_type
 
 
+def load_buffer_data_from_image(
+    path: Union[str, PathLike[str]],
+    flip_y: bool = False,
+    linearize: bool = False,
+    scale: float = 1.0,
+    offset: float = 0.0,
+    greyscale: bool = False,
+) -> np.ndarray[Any, Any]:
+    """
+    Helper to load an image from a file and convert it to a floating point tensor.
+    """ ""
+
+    # Load bitmap + convert to numpy array
+    bitmap = Bitmap(path)
+    data: np.ndarray[Any, Any] = np.asarray(bitmap)
+
+    # Validate array shape.
+    if data.ndim != 3:
```

Comment:
is this true for 2D/grayscale bitmaps too? don't remember if they are returned with dims=2